onnxruntime/core/session/inference_session.cc

```diff
@@ -1149,6 +1149,10 @@ void InferenceSession::AddPredefinedTransformers(GraphTransformerManager& transf
   if ((graph_optimization_level >= TransformerLevel::Level3) || !custom_list.empty()) {
     add_transformers(TransformerLevel::Level3);
   }
+
+  if ((graph_optimization_level >= TransformerLevel::Level4) || !custom_list.empty()) {
```

Comment:
Should TransformerLevel::Level4 be replaced with MaxLevel?

Reply:
No. Considering the case that someone add level 5 in the future.